src/Arcus.Messaging.ServiceBus.Core/Extensions/MessageExtensions.cs

```diff
@@ -93,9 +105,12 @@ public static string GetTransactionId(this Message message, string transactionId
             Guard.NotNull(message, nameof(message));
             Guard.NotNullOrWhitespace(transactionIdPropertyName, nameof(transactionIdPropertyName));
 
-            return message.UserProperties.TryGetValue(transactionIdPropertyName, out object transactionId)
-                ? transactionId.ToString()
-                : string.Empty;
+            if (message.UserProperties.TryGetValue(transactionIdPropertyName, out object transactionId))
+            {
+                return transactionId.ToString();
+            }
+
+            return string.Empty;
```

Comment:
Not really a fan returning an empty string here, can this be `null` instead?

Reply:
Sure